aip/0216.md

```diff
@@ -69,6 +69,22 @@ progression through a lifecycle.
 
 ## Additional Guidance
 
+### Default value
+
+The zero value of each state enum **should** follow a consistent convention:
```

Comment:
How about "the following convention" rather than "a consistent convention"? Otherwise it sounds like each API can make up its own.

Reply:
Done, but reworded a bit more to avoid "follow the following".